rotkehlchen/chain/ethereum/modules/eth2/eth2.py

```diff
@@ -258,19 +300,30 @@ def get_performance(
             to_query_indices = all_validator_indices
 
         if now - to_ts <= DAY_IN_SECONDS:
+            _, accumulating_validators = dbeth2.group_validators_by_type(
+                database=self.database,
+                validator_indices=to_query_indices,
+            )
             balances = self.beacon_inquirer.get_balances(
                 indices_or_pubkeys=list(to_query_indices),
                 has_premium=has_premium_check(self.premium),
             )
             for pubkey, balance in balances.items():
-                entry = pnls[pubkey_to_index[pubkey]]
+                entry = pnls[v_index := pubkey_to_index[pubkey]]
                 if 'exits' in entry:
                     continue  # no outstanding balance for exits
 
                 if balance.amount == ZERO:
                     continue
 
-                outstanding_pnl = balance.amount - FVal(32)
+                if v_index in accumulating_validators:
+                    if balance.amount < 2048:
```

Comment:
Define this somewhere as a constant. `MAX_EFFECTIVE_BALANCE`.

Also the `INITIAL_ETH_DEPOSIT` is wrong. As you can deposit as much as you want. But it starts counting only when it's above 32 which is now the `MIN_EFFECTIVE_BALANCE`

```suggestion
                    if balance.amount < 2048:
```